Source/WebKit/WebProcess/InjectedBundle/API/gtk/DOM/WebKitDOMRange.cpp

```diff
@@ -486,19 +486,7 @@ WebKitDOMDocumentFragment* webkit_dom_range_create_contextual_fragment(WebKitDOM
 
 gshort webkit_dom_range_compare_node(WebKitDOMRange* self, WebKitDOMNode* refNode, GError** error)
 {
-    WebCore::JSMainThreadNullState state;
-    g_return_val_if_fail(WEBKIT_DOM_IS_RANGE(self), 0);
-    g_return_val_if_fail(WEBKIT_DOM_IS_NODE(refNode), 0);
-    g_return_val_if_fail(!error || !*error, 0);
-    WebCore::Range* item = WebKit::core(self);
-    WebCore::Node* convertedRefNode = WebKit::core(refNode);
-    auto result = item->compareNode(*convertedRefNode);
-    if (result.hasException()) {
-        auto description = WebCore::DOMException::description(result.releaseException().code());
-        g_set_error_literal(error, g_quark_from_string("WEBKIT_DOM"), description.legacyCode, description.name);
-        return 0;
-    }
-    return result.releaseReturnValue();
+    return NULL;
```

Comment:
Why are we returning NULL? We should return 0 instead.

Reply:
Sure, will update. :-)

Reply:
Done in [3183fdb](https://github.com/WebKit/WebKit/commit/3183fdb64fb2abafc508f458ba691dbebaf567c9).